aten/src/ATen/native/Gesv.cpp

```diff
@@ -72,6 +72,34 @@ static void applyGesv(Tensor& b, Tensor& A, std::vector<int64_t> infos) {
   }
 }
 
+std::tuple<Tensor&,Tensor&> _gesv_single_out_cpu(
+    Tensor& result0, Tensor& result1,
+    const Tensor& self, const Tensor& A) {
+#ifndef USE_LAPACK
+  AT_ERROR("gesv: LAPACK library not found in compilation");
+#endif
+  int64_t bx = self.size(0);
+  int64_t by = (self.dim() == 1) ? 1 : self.size(1);
+  int info = 0;
+
+  result1 = A.t().clone();
```

Comment:
C++ question: does this reassign the reference `result1` to the output of `A.t().clone()` or does this "copy" `A.t().clone()` into `result1`?

Reply:
I discussed and this code is not right.

Reply:
I discussed this with @ezyang, it is the former -- I'll correct it in the next revision to check sizes and copy from the input matrices inplace